### PR TITLE
[LOOP-1634] AER reporting via mobile safari (feature)

### PR DESCRIPTION
**Pull request by nhamming**
https://tidepool.atlassian.net/browse/LOOP-1634

Adds the adverse event report in the support view. This will send the user to model safari and provide ZenDesk with field values using URL query parameters.

### CODE REVIEW

File: Loop/Views/SupportScreenView.swift [hunk 1]
```diff
@@ -33,10 +39,33 @@ struct SupportScreenView: View, HorizontalSizeClassOverride {
         .navigationBarTitle(Text("Support", comment: "Support screen title"))
         .environment(\.horizontalSizeClass, horizontalOverride)
     }
+    
+    private var adverseEventReport: some View {
+        Button(action: {
+            guard let url = self.adverseEventReportViewModel.reportURL else {
```
Comment (rickpasetto) [inline Loop/Views/SupportScreenView.swift:45]: [nit] It's nice that you did this, but wouldn't it be a code configuration (compile-time) error if `reportURL` is `nil`?  Seems unlikely this URL would be invalid...

File: Loop/View Models/AdverseEventReportViewModel.swift [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+import UIKit
+import HealthKit
+import LoopKit
+
+public class AdverseEventReportViewModel {
+    private let pumpStatus: PumpManagerStatus?
+
+    private let cgmDevice: HKDevice?
+
+    let formURLString = "https://support.tidepool.org/hc/en-us/requests/new?ticket_form_id=360000551951"
```
Comment (rickpasetto) [inline Loop/View Models/AdverseEventReportViewModel.swift:18]: I presume this URL (and the parameters) are valid?

**Review by rickpasetto — APPROVED** [review]
One nit, other than that LGTM.

File: Loop/Views/SupportScreenView.swift (hunk 1 above)
Comment (nhamming) [reply Loop/Views/SupportScreenView.swift:45]: That or something slips through by mistake. I guess I was thinking what would be worse, the user tapping the cell and nothing happening or it presents an alert. I chose the later.

File: Loop/View Models/AdverseEventReportViewModel.swift (hunk 2 above)
Comment (nhamming) [reply Loop/View Models/AdverseEventReportViewModel.swift:18]: Yes. Nick and I have been testing this and he is happy with the results.

**Pull request merged**